great_expectations/data_context/types/base.py

```diff
@@ -1184,6 +1184,7 @@ class DataContextConfigSchema(Schema):
     validations_store_name = fields.Str()
     evaluation_parameter_store_name = fields.Str()
     checkpoint_store_name = fields.Str(required=False, allow_none=True)
+    profiler_store_name = fields.Str(required=False, allow_none=True)
     plugins_directory = fields.Str(allow_none=True)
```

Comment:
This entire file is just ensuring parity between the new ProfilerStore and the older stores (in terms of DataContext config)